docs/source/pages/output_files.rst

```diff
@@ -41,17 +41,35 @@ The report files of Macaron (from using the :ref:`analyze action <analyze-action
 Unique result path
 ''''''''''''''''''
 
-For each target repository, Macaron creates a directory under ``reports`` to store the report files. This directory
-path is formed from the git host name (e.g ``github.com``), the owner and the name of that
-repository. The final path is created using the following template:
+For each target software component, Macaron creates a directory under ``reports`` to store the report files. This directory
+path is formed from the PURL string of that component. The final path is created using the following template:
 
 .. code-block::
 
-    <path_to_output>/reports/<git_service_name>/<owner>/<repo_name>
+    <path_to_output>/reports/<purl_type>/<purl_namespace>/<purl_name>
 
-.. note:: The git host name has all occurrence of ``.`` in the URL replaced by ``_``.
+For more information on the three fields ``type``, ``namespace`` and ``name`` of a PURL string, please see
+`PURL Specification <https://github.com/package-url/purl-spec/blob/master/PURL-SPECIFICATION.rst>`_.
 
-For example, the reports for `<https://github.com/micronaut-projects/micronaut-core>`_ repository will be stored under
+Typically, when a repository path is provided as the main software component of the :ref:`analyze action <analyze-action-cli>`,
+a PURL is generated from the repository path, which is then later used in generating the unique report path.
+
+For example, when run this command:
```

Comment:
```suggestion
For example, when running this command:
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/401/commits/7a84f6f91fe54da57436ef5ac9e065a913932769